ios/README.md

```
@@ -60,24 +60,19 @@
+- **Apple Strings**: Native iOS `.strings` files (e.g., `Localizable.strings`) exported directly into the Xcode project structure.
 
 ### Moko
 
-- Error messages are shared via [Moko Resources](https://github.com/icerockdev/moko-resources), so
-  that we can use the strings in the shared code and avoid duplicities when converting errors to
-  string messages
-- Error strings are stored in the `twine/errors.txt` file
-- Script `generate-error-messages.sh` calls needed gradle tasks (`generateErrorsTwine`
-  and `generateMRCommonMain`) to generate `MR` class
+Strings are shared via [Moko Resources](https://github.com/icerockdev/moko-resources), so
+that we can use the strings in the shared code 
```

markdownlint-cli2 (0.21.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)